core/src/main/java/org/bitcoinj/script/ScriptChunk.java

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright 2013 Google Inc.
  * Copyright 2014 Andreas Schildbach
+ * Copyright 2017 John L. Jegutanis
```

Comment:
Since you removed your change to ScriptChunk, can you remove this copyright addition too?

Reply:
Removed